test/hotspot/jtreg/vmTestbase/nsk/jdi/VirtualMachine/dispose/dispose002a.java

```diff
@@ -127,14 +127,14 @@ public static void main (String argv[]) {
                          while (true) {
                              instruction = pipe.readln();
                              if (instruction.equals("check_done")) {
-                                 if (test_thread.isAlive()) {
+                                 if (Utils.isAlive(test_thread)) {
```

Comment:
We also have `jdk.test.lib.Utils` being referenced as `Utils` in the main test. Maybe `JDIUtils` would be better.

Also, `isAlive` is a bit generic sounding. It would be nice if the name conveyed the relationship to waiting after a dispose. Maybe `isAliveAfterVMDispose`. At least then the user knows there's something special about the "is alive" check being made.

Reply:
Actually `isAliveAfterVMDispose` conveys the wrong message. The idea is to give the `VM.dispose()` a chance to happen. Maybe `isAlive` should just better document why it waits.

Reply:
Or you could use the longer name `nsk.share.jdi.Utils` to differentiate...

Is it always the case that we're calling this `isAlive()` function after a `VM.dispose()` call?
I'm not so sure that's the case... See L135-142 below. That looks like a simple query response
and not necessarily after a `VM.dispose()` call...

Reply:
I updated it to JDIUtils.waitForCompletion(Thread thread) to be more clear about what we are doing here.